samples/bluetooth/central_and_peripheral_hr/README.rst

```diff
@@ -33,24 +33,22 @@ The sample supports the following development kits:
    :header: heading
    :rows: nrf5340dk_nrf5340_cpuapp_and_cpuapp_ns, nrf52840dk_nrf52840, nrf52dk_nrf52832, nrf52833dk_nrf52833, nrf52833dk_nrf52820, nrf21540dk_nrf52840
 
-Bluetooth® LE Central Role only:
+For testing just the Bluetooth® LE Central Role operation the sample requires one of the following setups:
```

Comment:
```suggestion
To test just the Bluetooth® LE Central Role operation, you need the following setup:
```

Reply:
Fixed in a bit different way to emphasize the bullet list is an alternative.

Reply:
Right, I misunderstood this. I thought the alternatives were a smartphone or tablet AND another development kit. Now that you corrected me, I think the original suggestion was better than this.

Reply:
done